ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.ts

```diff
@@ -72,25 +78,33 @@ export class IndividualJourney extends JourneyBase {
     }
 
     let nextStep = this.stepOrder[currentStep + 1];
+    this.updateSubmitModelWithStep(currentStep);
 
     // access to a computer.
     if (this.model.computer === false) {
```

Comment:
This method is pretty big including this, any chance you can break it out?

Reply:
this check is moved to the submit service.